.github/workflows/ci.yml

```
@@ -20,6 +20,7 @@ on:
 env:
   CARGO_INCREMENTAL: 0
   RUST_BACKTRACE: full
+  COLORBT_SHOW_HIDDEN: '1'
```

I just checked the docs, and we need this too:
https://docs.rs/color-eyre/latest/color_eyre/#multiple-report-format-verbosity-levels
https://doc.rust-lang.org/std/backtrace/index.html#environment-variables

```suggestion
  RUST_BACKTRACE: full
  RUST_LIB_BACKTRACE: full
  COLORBT_SHOW_HIDDEN: '1'
```